Node Status Monitor/Python v2.7/api_helper.py

```diff
@@ -0,0 +1,111 @@
+import requests
```

Comment:
Please be sure to follow the python coding standards.

Reply:
done